v3/public-resolvers.md

```diff
@@ -573,16 +598,7 @@ ControlD Free DNS. Take CONTROL of your Internet. Block unwanted content, bypass
 
 This DNS unblocks censored domains from various countries.
 
-sdns://AgcAAAAAAAAACjc2Ljc2LjIuMTEAFGZyZWVkbnMuY29udHJvbGQuY29tCy91bmNlbnNvcmVk
-
-
-## controld-unfiltered
-
-ControlD Free DNS. Take CONTROL of your Internet. Block unwanted content, bypass geo-restrictions and be more productive. - https://controld.com/free-dns
-
-This is a Unfiltered DNS, no DNS record blocking or manipulation here, if you want to block Malware, Ads & Tracking, or Social Network domains, use the other ControlD DNS configs.
-
```

Comment:
I don't think it's a good idea to also change to order of resolvers here.

Reply:
I think it is better to leave the official order that the dns have on their website:

1. First without filter
2. After with filters
3. From last uncensored.

Reply:
I think it is better to leave the official order that the dns have on their website: https://controld.com/free-dns

1. First without filter.
2. After with filters.
3. From last "uncensored".

Reply:
I have already moved the DNS as you recommend.